AzureCalling/Controllers/LobbyViewController.swift

```diff
@@ -75,6 +75,21 @@ class LobbyViewController: UIViewController, UITextFieldDelegate {
         return true
     }
 
+    func resetRendererView() {
+        callingContext.withLocalVideoStream { [weak self] localVideoStream in
+            DispatchQueue.main.asyncAfter(deadline: .now() + 1) { [weak self] in
```

Comment:
Why do we need to asyncAfter 1 second?

Reply:
It's due to the Calling SDK `VideoStreamRenderer` unavailable for about 0.5 seconds when call ended while video stream is on